README.md

```diff
@@ -35,6 +47,11 @@ Example Playbook
 - hosts: reboot-servers
   roles:
     - role: oasis-roles.reboot
+      reboot_seconds_before_reboot: "2"
```

Comment:
Quotes shouldn't really be necessary here, since interpreting as a string or number are basically interchangeable here.